third_party/move/move-vm/types/src/values/values_impl.rs

```diff
@@ -2307,6 +2307,25 @@ fn check_elem_layout(ty: &Type, v: &Container) -> PartialVMResult<()> {
 }
 
 impl VectorRef {
+    pub fn len_usize_raw(&self, type_param: &Type) -> PartialVMResult<usize> {
```

Comment:
I've added this to not need to wrap / unwrap / cast for getting the length inside of native function.